fixtures/errors/src/errors.udl

```diff
@@ -27,6 +27,11 @@ interface ValidationError {
   UnknownError();
 };
 
+[Error]
+interface ErrorNamedError {
+  Error(string error);
```

Comment:
Could you reference this field in tests? That would ensure that this definition is not removed from UDL file be accident, which would not be detected in any way without a test referencing it.